app/views/admin/classifications/_form.html.erb

```diff
@@ -12,26 +12,14 @@
   </fieldset>
 
   <fieldset>
+    <%= render "admin/editions/related_policy_fields", form: classification_form %>
```

Comment:
If this partial is now being used for non-edition models, I think it makes sense to move it out of `views/admin/editions` into somewhere more appropriate, e.g. `views/admin/shared`.